src/globals/variables.scss

```diff
@@ -21,7 +21,13 @@ $colour-tertiary-coral: #FF7272;
 
 $colour-brand-cc-red: #B30510; // Christmas Challenge
 $colour-brand-wgmf-purple: #6E0887; // Women and Girls Match Fund
-$colour-brand-gmf-green: #50B400; // Green Match Fund
+
+$colour-brand-gmf-green: #50B400; // Green Match Fund - deprecated name
+$colour-brand-er-green: #50B400; // Earth Raise
+$colour-brand-er-blue: #0c0068;
```

Comment:
Maybe slightly more consistent with the original ticket description to call this dark if we're doing so with the green? Not a blocker.

```suggestion
$colour-brand-er-dark-blue: #0c0068;
```

Reply:
yep the colour is this ![image](https://github.com/user-attachments/assets/7d85a014-18e7-4ee6-9dd9-925d280e71df), I think dark blue is a better name so we know without having to see it rendered that it needs a light colour for contrast. Will change and merge.